examples/regular-expressions/regular_expressions.md

```diff
@@ -4,6 +4,8 @@ The `regex` library provides functionalities for searching, splitting, and repla
 
 For more information on the underlying module, see the [`regex` module](https://lib.ballerina.io/ballerina/regex/latest/).
 
+**Deprecation Notice:** The ``regex`` standard library is deprecated and will no longer be maintained or updated. We recommend that you migrate to [ballerina/lang.regexp](https://lib.ballerina.io/ballerina/lang.regexp/latest) library for continued support and updates.
```

Comment:
```suggestion
**Deprecation Notice:** The ``regex`` standard library is deprecated and will no longer be maintained or updated. We recommend that you migrate to [ballerina/lang.regexp](https://lib.ballerina.io/ballerina/lang.regexp/latest) library for continued support and updates.
```
Change this to the below as per https://github.com/ballerina-platform/module-ballerina-regex/pull/411:

Praneesha Chandrasiri, Yesterday 12:29 PM, Edited
**Deprecation Notice:** This library is deprecated and will no longer be maintained or updated. Instead, it is recommended to use the [ballerina/lang.regexp](https://lib.ballerina.io/ballerina/lang.regexp/latest) library for continued support and updates.